keyboards/handwired/illera16/info.json

```diff
@@ -0,0 +1,48 @@
+{
+    "manufacturer": "victor_illera",
+    "keyboard_name": "illera16",
+    "maintainer": "villeramassana",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Comment:
```suggestion
        "nkro": true,
        "encoder": true,
        "rgblight": true
```

Reply:
i cant commit it because its outdated, how can I proceed to commit it?

Reply:
done!